.github/workflows/autobuild.yml

```diff
@@ -251,7 +251,7 @@ jobs:
           - config_name:            iOS (artifacts)
             target_os:              ios
             building_on_os:         macos-14
-            base_command:           QT_VERSION=6.7.3 ./.github/autobuild/ios.sh
+            base_command:           QT_VERSION=6.9.1 ./.github/autobuild/ios.sh
```

Comment:
So if we merge this with `QT_VERSION` here still `6.7.3`, will that pin the version or is something else needed?

Reply:
I mean will github-actions stop pushing version bumps.

Reply:
OK, so we need a PR with that in, too, I guess.